reporter/lib/events.js

```diff
@@ -0,0 +1,50 @@
+const io = require('socket.io-client');
+const { nanoid } = require('nanoid');
+
+const socket = io('http://localhost:5001/');
+const processId = nanoid(16);
+
+module.exports = {
+    handleStart: (err, args) => {
+        if (err) {
+            // return if any error on starting the run
+            return;
+        }
+
+        socket.emit('control:new-run', {
+            processId,
+        });
+    },
+
+    handleDone: (err, args) => {
+        if (err) return;
+
+        socket.emit('control:end-run', {
```

Comment:
How about sending the events as-is from newman. e.g. `done` event can be sent as `done` itself? Any specific reason to choose `control:end-run`?

Reply:
Would take care of this in the pub/sub PR!